src/Tablet/Sensors/SensorsTab.js

```diff
@@ -96,28 +97,28 @@ const SensorsTab = () => {
 
   const renderCard = type => {
     return (
-      <Col sm={4} style={{ marginTop: 20 }}>
-        <Card>
-          <Card.Header>{type.toUpperCase()}</Card.Header>
-          <Card.Body>{renderType(type)}</Card.Body>
-        </Card>
-      </Col>
+      <IonCol sm={4} style={{ marginTop: 20 }}>
```

Comment:
`IonCol` doesn't have `sm` property
https://ionicframework.com/docs/api/col

Reply:
So should the Col stay as Col instead of IonCol?

Reply:
you can just remove the `sm` prop